### ISSUE TITLE
Add free map material without modern-day geopolitical and infrastructural features

### ISSUE THREAD

**Issue by mfranke93** [UniStuttgart-VISUS/damast#224]
The default OSM map tiles are nice, but do not fit the historical data very well regarding borders, roads, etc.

**Comment by Schoeneh** [comment]
@mfranke93 Hi!
I'm the SHK (student assistant) of Dorothea Weltecke, who also dabbles in GIS and the Spatial Humanities.
Maybe we could talk about possible base maps / map tiles?
(I've done some work with [naturalearthdata.com](https://www.naturalearthdata.com/) - especially Natural Earth 2 - over the last few years...)

**Comment by mfranke93** [comment]
Hi, nice to meet you! Sure, we can do that :smiley: 

As it happens, I was just working on this issue as your mail arrived. Over the last two months, I've created a set of map tiles that just show shaded relief and no other features whatsoever, because I needed permissive material for the screenshots in my thesis as well.

I have found that just the shaded relief; in combination with oceans, lakes, and rivers from Natural Earth; already provide a good background map for Damast that does not distract too much from the visualization in the foreground (see attached image). 

![image](https://github.com/UniStuttgart-VISUS/damast/assets/9373678/0d6d20be-02d5-4112-88e3-e58f22112c65)

I'm quite close to merging this into `main` and deploying to the instances at HU. But that doesn't mean we can't add other map material as well.
